general/multi_companies/manage_multi_companies.rst

```diff
@@ -0,0 +1,134 @@
+- *Adding a company*: the record is visible to users logged in to that specific company and its child
+  companies.
+
+.. image:: media/product_form_company.png
+   :align: center
+   :alt: View of a product's form emphasizing the company field in Odoo Sales
+
+Employees' access
+-----------------
+
+Once companies are created, manage your employees' :doc:`Access Rights <../odoo_basics/add_user>`
+for *Multi Companies*.
+
+.. image:: media/access_rights_multi_companies.png
+   :align: center
+   :alt: View of an user form emphasizing the multi companies field under the access rights tabs
+         in Odoo
+
+| If a user has multiple companies *activated* on his database, and he is **editing** a record,
+  the editing happens on the record's related company.
+| Example: if editing a sale order issued under JS Store US while working on the JS Store Belgium
+  environment, the changes are applied under JS Store US (the company from which the sale order
+  was issued).
+| When **creating** a record, the company taken into account is:
+
+- The current company (the one active) or,
+- No company is set (on products and contacts’ forms for example) or,
+- The company set is the one linked to the document (the same as if a record is being edited).
+
+Documents’ format
+-----------------
+
+To set documents' formats according to each company, *activate* and *select* the respective one and,
+under *Settings*, click on *Configure Document Layout*.
+
+.. image:: media/document_layout.png
+   :align: center
+   :alt: View of the settings page emphasizing the document layout field in Odoo
+
+Inter-Company Transactions
+--------------------------
+
+First, make sure each one of your companies is properly set in relation to:
+
+- :doc:`Chart of Accounts <../../accounting/overview/getting_started/chart_of_accounts>`
+- :doc:`Taxes <../../accounting/fiscality/taxes/default_taxes>`
+- :doc:`Fiscal Localizations <../../accounting/fiscal_localizations/overview/fiscal_localization_packages>`
+- :doc:`Pricelists <../../sales/products_prices/prices/pricing>`
+- :doc:`Warehouses <../../inventory/management/warehouses/warehouse_creation>`
+
+.. LINK JOURNAL AND FISCAL POSITIONS DOCS ONCE AVAILABLE + DELETE NOTE.
+
+Now, activate the *Inter-Company Transactions* option under *Settings*. With the respective company
+*activated* and *selected*, choose if you would like operations between companies to be synchronized
+at an invoice/bills level, or, at a sales/purchase orders level.
+
+.. image:: media/inter_company_transactions.png
+   :align: center
+   :alt: View of the settings page emphasizing the inter company transaction field in Odoo
+
+- **Synchronize invoice/bills**: generates a bill/invoice when a company confirms a bill/invoice for
+  the selected company.
+
+  *Example:* an invoice posted on JS Store Belgium for JS Store US, automatically creates a vendor
+  bill on the JS Store US from the JS Store Belgium.
```

Comment:
```suggestion
  *Example:* an invoice posted on JS Store Belgium for JS Store US automatically creates a vendor
  bill on the JS Store US from the JS Store Belgium.
```

Reply:
I will keep the comma to give readers a break-time to absorb the 'on' and 'from' info ^^ (:

Reply:
Yeah, no problem :)
Then, can I suggest this:

```suggestion
  *Example:* an invoice posted on JS Store Belgium, for JS Store US, automatically creates a vendor
  bill on the JS Store US, from the JS Store Belgium.
```